ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/components/TalkCard.kt

```diff
@@ -377,8 +391,17 @@ private fun FeedbackBlock(
                                 if (userSignedIn) {
```

Comment:
should we extract extension?

Reply:
I'm not sure which part you mean, but we can do it later :)